lib/riffed/server.ex

```diff
@@ -206,13 +218,14 @@ defmodule Riffed.Server do
     after_start = Module.get_attribute(env.module, :after_start) || quote do: fn (_, _) -> nil end
     error_handler = Module.get_attribute(env.module, :error_handler) |> build_error_handler
 
+    are_structs_auto_imported = Module.get_attribute(env.module, :auto_import_structs) or false
     function_handlers = Enum.map(functions,
       fn({fn_name, delegate}) ->
-        build_handler(thrift_meta, struct_module, fn_name, delegate, overrides.functions) end)
+        build_handler(thrift_meta, struct_module, fn_name, delegate, overrides.functions, are_structs_auto_imported) end)
 
     structs_keyword = ThriftMeta.Meta.structs_to_keyword(thrift_meta)
 
-    struct_module = if Module.get_attribute(env.module, :auto_import_structs)  do
+    struct_module = if are_structs_auto_imported do
```

Comment:
Would read better if this was named `struct_are_auto_imported` or `auto_imported_structs`.